# Poster Abstracts {-}
## Multi-Environment Trial Analysis Of Agronomy Trials Using Established Plant Population Density {-}
<p style="text-align:center">
Michael Mumford and Kerry Bell<br />
Queensland Department of Agriculture and Fisheries<br />
</p>
There is a growing interest within the grains industry to determine the yield response to plant population density. This can vary depending on the trial location, the hybrid that is planted and the management practices that are employed.

In the past, the analysis of plant population density has been performed using the targeted plant population density as determined by the number of seeds initially planted. However, it has been found that the targeted plant population density may differ significantly from the established plant population density, i.e. the plant population density observed in the field. In such circumstances, it becomes important to use the established rather than targeted plant population densities in order to provide the most informative measure of yield response to plant density. This is especially true when performing a multi-environment trial (MET) analysis due to variation between the targeted and established plant population density at each trial.

We propose a procedure for performing a MET analysis using established plant population density as an explanatory variable. Using a linear mixed model framework, separate regression lines/curves are fit to each combination of trial and management practice, which we define as ‘environment’. Environments are then grouped according to the results of a cluster analysis performed using the estimates of the regression coefficients, ensuring that environments within a cluster share a similar response to plant density. The lines/curves fitted to environments within clusters are then parallel, allowing us to then perform Fisher’s least significant difference testing for yield differences between environments within each cluster.

This presentation will focus on the application of the proposed method to a large set of maize and sorghum agronomy trials conducted across New South Wales and Queensland over the last three years. For simplicity, a separate analysis was performed for each hybrid.
<p class="pagebreak"></p>
## Adventures In Digital Agriculture In New Zealand {-}
<p style="text-align:center">
Esther Meenken and Vanessa Cave<br />
AgResearch<br />
</p>
New Zealand is a nation for which primary industries are the economic backbone. New Zealand's highly variable landscape means there are a wide range in climatic and soil conditions, with multiple land uses existing in relatively small spatial areas. This means there are many possibilities to be explored in Digital Agriculture as advances in sensory technologies are made. The technical challenges known to be related to the use of sensors include issues around big data, missing data, biased and variable sensors, and various types of structured and non-normal data. We will describe a suite of studies that have been recently undertaken that can be loosely grouped under the Digital Agriculture umbrella. Case studies explore a range of scientific and statistical challenges in the primary industries arena, and include the use of classical and Bayesian Principled Experimental Design, Hierarchical Bayesian Models, neural net classifiers, visualisation, animal tracking via GPS, and data management.
<p class="pagebreak"></p>
## Modelling Canopy Greenness Over Time Using Splines And Non-Linear Regression {-}
<p style="text-align:center">
Bethany Macdonald^1^, Jack Christopher^2^, and Alison Kelly^1^<br />
^1^Queensland Department of Agriculture and Fisheries<br />
^2^Queensland Alliance for Agriculture and Food<br />
</p>
The ability of a plant to retain leaf greenness for an extended time after anthesis, known as stay-green, has been linked to greater yield in wheat and can vary amongst genotypes. Many aspects of a plant's senescence contribute to stay-green; therefore, accurately modelling senescence and understanding the genetic variation in senescence dynamics is integral to selecting for the stay-green phenotype. Normalised difference vegetative index (NDVI) provides a measure of canopy greenness and when collected over time, can give an indication of when senescence begins and the rate at which it continues. 
 Christopher et al. (2014) captured genetic variation in senescence dynamics using multiple stay-green traits derived from a logistic regression of longitudinal NDVI measurements. These traits were estimated independently for each plot within the experiment, ignoring any sources of covariance, and then subsequently analysed using a linear mixed model. However, no estimation errors were carried from the first to the second stage of the analysis.

 We discuss two alternative one-stage methods for modelling senescence dynamics based on longitudinal NDVI measurements. Both of these methods enable the variability in senescence patterns to be partitioned into genetic and non-genetic components, while also incorporating the experimental structure. The first method involves the use of splines in a linear mixed model framework. This method allows an appropriate covariance structure for repeated measures to be utilised and, unlike other approaches, enables flexibility in the senescence patterns. The second method involves fitting logistic equations in a non-linear mixed model framework. The estimated parameters have biological interpretations, aiding in the genetic comparison of senescence dynamics. These two methods offer a more statistically robust approach to modelling crop senescence and the underlying genetic variability.
<p class="pagebreak"></p>
## On Testing Marginal Homogeneity For Square Contingency Tables With Ordinal Categories {-}
<p style="text-align:center">
Kouji Tahata<br />
Tokyo University of Science<br />
</p>
For the analysis of square contingency tables with ordered categories, the marginal homogeneity model, which indicates the row marginal distribution is equal to the column marginal distribution, were considered. Stuart (1955) proposed the test statistics (denoted by Q) for the marginal homogeneity model. However it does not use the information of the category ordering. Agresti (1983) compared between Q and the Mann-Whitney test about the power by simulations. In this paper, we consider some measures to represent the degree of departure from the marginal homogeneity. For example, Tomizawa, Miyamoto and Ashihara (2003) and Tahata, Iwashita and Tomizawa (2008). In the situation given by Agresti (1983), we compare between Q and test based on these measures about the power.
<p class="pagebreak"></p>
## A Factor Analytic Approach To Modelling Disease Progression Across Leaf Layers And Time {-}
<p style="text-align:center">
Clayton Forknall, Greg Platz, Lisle Snyman, and Alison Kelly<br />
Queensland Department of Agriculture and Fisheries<br />
</p>
Controlling and limiting the impact of foliar diseases is a challenge faced by the Australian grains industry. Foliar diseases compromise and destroy photosynthetic area, thus limiting plant resources and adversely affecting crop productivity. Such diseases often infect the lower leaf layers of plants early in the season and, dependent on their compatibility with the host and environment, progress towards the topmost leaf layers over time. 

 Synthesizing the complicated dynamics of disease progression, over different leaf layers and across a growing season, into a measure to estimate the impact of loss of leaf area on productivity requires the assessment of the proportion of leaf area compromised by disease (LAD) at multiple times throughout the season. A simple measure that captures both LAD and disease duration on a given leaf layer is the Area Under the Disease Progress Curve (AUDPC), formed by applying the trapezoid rule to LAD assessments over time. The AUDPC is then often correlated to a measure of productivity for estimating the impact of disease at either a variety or experimental unit level.

 We propose an alternative approach to the AUDPC to model disease progression over time and leaf layers more efficiently. Using a factor analytic approach, the covariance between leaf layers, both within and across assessment times, is captured on a variety basis in a linear mixed model framework. This approach will not only provide more reliable estimates of the disease pressure exerted on varieties, but also inform of relationships between leaf layers and how consistent these relationships are across time.
 Better quality information regarding the relationship between loss of leaf area and productivity can be made available to industry through the improved estimates and understanding of disease progression obtained from this modelling approach.
<p class="pagebreak"></p>
## Associating Straw Strength With Likelihood Of Head Loss In Barley For Western Australia {-}
<p style="text-align:center">
Dean Diepeveen^1^, Kefei Chen^1^, Chengdao Li^2^, and David Farleigh^3^<br />
^1^Curtin University<br />
^2^Murdoch University<br />
^3^Department of Primary Industries and Development<br />
</p>
Barley headloss is characterised by straw breakage just below the head at or near plant maturity. Our recent research into barley headloss has enabled us to evaluate the various components of plant maturity and have found a clear genetic component to this industry issue. The more challenging components of this research is to understand why our more tolerant cultivars also show substantial headloss in certain unseasonal conditions during harvest. Our preliminary analyses uses environmental data with plant measurement data and looks at the incidences of extreme weather events on barley headloss. What confounds the analysis is the nutrient/growth conditions of the plant and the predisposition of these barley crops to headloss during these weather events. This paper will provide some lessons learnt and future directions.
<p class="pagebreak"></p>
## A Deep Learning Neural Network Model To Identify The Important Genes In Metastatic Breast Cancer From Censored Microarray Data {-}
<p style="text-align:center">
Quoc-Anh Trinh<br />
Vietnam National University<br />
</p>
Microarray data have been shown to correlated with survival in breast cancer. The most difficult in analysis of this data consist in a simultaneous measure of huge gene expression levels over a few patients available. The conventional survival models such as the proportional hazard model of Cox are no more appropriated including the hazard proportional hypothesis and the linear covariates effect hypothesis. Prognostic studies involve the identification of genes that correlate with survival in order to provide new information on pathogenesis. This result may aid in the research of drug design for new targets.
 We introduce in this paper a deep learning recurrent neural network approach to predict survival times for the individual patient based on microarray measurement. This neural network defines the hazard function of survival not to mention the proportionality and linearity hypothesis. We present a deep learning approach to the identification of genes critical for the metastases of breast cancer. We identified a set of highly interactive genes by analysing the connectivity matrices.
<p class="pagebreak"></p>
## Using R And Shiny To Develop Web Based Sampling Applications For The Agricultural And Education Sectors {-}
<p style="text-align:center">
Peter Kasprzak, Olena Kravchuk, and Andy Timmins<br />
Univeristy of Adelaide<br />
</p>
Sampling is an important aspect of agricultural statistics and an important feature of industrial reality. Enabling an efficient dialogue with researchers and agronomists about sound methods for field data collection will increase the efficiency and reliability of agricultural projects. The Shiny package in R allows interactive web applications to be created based upon existing R packages. This presentation will outline the steps involved in creating an app that implements sampling techniques such as SRS, Stratified Sampling, and Ratio Sampling, within a visually appealing graphical environment for use in the education and agricultural sectors.
<p class="pagebreak"></p>
## Graphical Network Analyses Informs Biomarker Discovery Via Quantification Of Key Disease Related Connections {-}
<p style="text-align:center">
James Doecke<br />
CSIRO<br />
</p>
Determination of the optimal set of biomarkers often relies on large scale feature selection, that selects a minimal set of independent biomarkers, without consideration of any information regarding the biological network. This research is aimed at defining biological network structures related to disease phenotype, through a three-level statistical process. Briefly, we first implement an unsupervised correlation structure across the set of biomarkers, used to define clusters of biomarkers via strict thresholding. Next, using the graphical network analysis method Differential Network Analysis in Genomics (DINGO), we define the differential network structure for healthy and disease states. Lastly from the derived differential connectivity score we cluster the individual groups of biomarkers. From this process, we are able to produce a network of similarly grouped sets of biomarkers that can separate healthy from disease subjects. Lastly we perform a random effects model for each cluster to estimate the association with disease status. Using this design, we have been able to incorporate information from multiple correlated biomarkers to increase the proportion of variance explained in disease status. We test this design on two data sets, 1) a set of ~2000 peptide biomarkers, with the express aim of defining sets of biomarkers associated with Alzheimer’s Disease pathology, and 2) a set of ~2000 mRNA probe sets combined with a set of ~2000 DNA methylation probes with the express aim of defining sets of biomarkers associated with breast cancer. We present network plots and connectivity structures for each data set, and highlight the increased performance of clustered biomarkers as compared to individual biomarkers to separate disease status. Lastly, we show that by utilizing a network approach, we can incorporate more biological information from the data to explain outcome.
<p class="pagebreak"></p>
## Spatio-Temporal Cortical Brain Atrophy Patterns Of Alzheimer's Disease {-}
<p style="text-align:center">
Marcela Cespedes^1^, James McGree^1^, Christopher Drovandi^1^, Kerrie Mengersen^2^, James Doecke^3^, and Jurgen Fripp^3^<br />
^1^Queensland University of Technology<br />
^2^Queensland University of Technlogy<br />
^3^CSIRO<br />
</p>
The degeneration of the cerebral cortex is a complex process which often spans decades. This degeneration can be evaluated on regions of interest (ROI) in the brain through probabilistic network analysis. However, current approaches for finding such networks have the following limitations: 1) analysis at discrete age groups cannot appropriately account for connectivity dynamics over time; and 2) morphological tissue changes are seldom unified with networks, despite known dependencies. To overcome these limitations, a probabilistic dynamic wombled model is proposed to simultaneously estimate ROI cortical thickness and network continuously over age, and was compared to an age aggregated model. The inclusion of age in the network model was motivated by the interest in investigating the point in time when connections alter as well as the length of time required for changes to occur. Our method was validated via a simulation study, and applied to healthy controls (HC) and clinically diagnosed Alzheimer's disease (AD) groups. The probability of a link between the middle temporal (a key AD region) and the posterior cingulate gyrus decreased from age 55 (posterior probability > 0.9), and was absent by age 70 in the AD network (posterior probability < 0.12). The same connection in the HC network remained present throughout ages 55 to 95 (posterior probability &ge; 0.75). The analyses presented in this work will help practitioners choose suitable statistical methods to identify key points in time when brain covariance connections change, in addition to morphological tissue estimates, which could potentially allow for more targeted therapeutic interventions.
<p class="pagebreak"></p>
## Empirical Modelling Of Fruit Firmness Change During Colour Conditioning {-}
<p style="text-align:center">
Lindy Guo and Ringo Feng<br />
Plant and Food<br />
</p>
Gold3 (Actinidia. chinensis *Zesy002*, marketed as Zespri&reg; SunGold Kiwifruit) is a new yellow-fleshed kiwifruit cultivar being used to replace *Hort16A* that has been almost all destroyed by the outbreak of Pseudomonas syringae pv. actinidiae (Psa) in 2010. The Smart Monitoring programme started in 2011, records fruit development, maturation, harvesting and storage performance of Gold3 kiwifruit from 12 orchards in a fairly consistent manner year after year. 

The aim of this project is to summarize fruit development and storage potentials over the 5 years, and hence provide a better understanding of the roles of seasonal weather conditions, orchard management, and fruit attributes at harvest affecting storage potential. 

For this project, storage life was defined as storage time for a batch of fruit (fruit harvested from an orchard on a particular date) to soften to 1 kfg. Several ways to calculated storage life based on firmness monitoring data are compared and the storage lives were correlated with weather conditions, orchard management, and fruit attributes at harvest using different algorithms. The results will be presented to highlight experiences learnt in this data analysis. Recommendations will be made on dealing with multiyear data with collinear variables.
<p class="pagebreak"></p>
